LICENSE_MIT.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Tomorrow

```

Comment:
Should the date here include a time span, i.e. 2017-2022?

Reply:
Not sure if we should modify the text here as it's the license itself. According to our lawyers the date should be specified in the readme file

Reply:
Huh that's strange, I don't think I have ever seen a date for a license in a readme before. But I guess that works?

Reply:
Maybe adding 2020-2022 here since the original data in the license was 2020? and then specify the date in the readme would be a good combination of both options?

Reply:
I think that would work but please do double check with @madsnedergaard as well. 👍🏼

Reply:
I removed the date from the readme and only specified version+commit, let me know what you think about it

Reply:
I think date span makes sense, to make it clearer for people that this license was active during that time (but not before or after).